packages/nc-gui-v2/pages/index/index/create-external.vue

```diff
@@ -235,14 +298,7 @@ onMounted(() => {
   >
     <general-noco-icon class="color-transition hover:(ring ring-accent)" :class="[isLoading ? 'animated-bg-gradient' : '']" />
 
-    <div
-      class="color-transition transform group absolute top-5 left-5 text-4xl rounded-full bg-white cursor-pointer"
-      @click="navigateTo('/')"
-    >
-      <MdiChevronLeft class="text-black group-hover:(text-accent scale-110)" />
-    </div>
-
-    <h1 class="prose-2xl font-bold self-center my-4">{{ $t('activity.createProject') }}</h1>
```

Comment:
Why was this removed and replaced by a `h3`?
Now there's no navigation option in the form. 😢

Reply:
oh this should be by mistake, probably while rebasing. thank you for pointing out!